platforms/extensibility/plugin-use/src/main/java/org/gradle/plugin/use/resolve/internal/ArtifactRepositoriesPluginResolver.java

```diff
@@ -57,7 +55,7 @@ public ArtifactRepositoriesPluginResolver(DependencyResolutionServices resolutio
     public PluginResolutionResult resolve(PluginRequestInternal pluginRequest) {
         ModuleDependency markerDependency = getMarkerDependency(pluginRequest);
         String markerVersion = markerDependency.getVersion();
-        if (isNullOrEmpty(markerVersion)) {
+        if (Strings.isNullOrEmpty(markerVersion)) {
```

Comment:
this feels really silly. same like like `requireNonNull`. who needs the context of we have compilers around. 

imho, its poluting dry pattern `String Strings`.

imho its just about to avoid super generics like `of` and same non-saying therefore worth to give context.